src/server/services/tips/predicates.ts

```diff
@@ -32,6 +32,15 @@ export const hasValidId: TipsPredicateFN = (
   });
 };
 
+export const hasIdExpieringDuringPeak: TipsPredicateFN = (
+  appState,
+) => {
+  const ids = appState.BRP?.content?.identiteitsbewijzen ?? [];
+  return ids.some((idBewijs: Identiteitsbewijs) => {
+    const afloop = new Date(idBewijs.datumAfloop);
+    return new Date('2024-03-01') <= afloop && afloop < new Date('2024-09-01');

```

Comment:
Kun je hier constants van maken? `ID_BEWIJS_PEAK_DATE_START = new Date('2024-03-01')` en `ID_BEWIJS_PEAK_DATE_END = new Date('2024-09-01')` met comment wat peakdate start en end zijn. (betere naam mag ook. Bv `DATUM_START_PIEK_AANVRAAG_PERIODE`

Reply:
Done in de nieuwe